bot/modules/guild_admin/economy/set_coins.py

```diff
@@ -0,0 +1,104 @@
+import discord
+import datetime
+from cmdClient.checks import in_guild
+
+from settings import GuildSettings
+from core import Lion
+
+from ..module import module
+
+POSTGRES_INT_MAX = 2147483647
+
+@module.cmd(
+    "set_coins",
+    group="Guild Admin",
+    desc="Set coins on a member."
+)
+@in_guild()
```

Comment:
Can you add a `guild_admin` ward here please!

Reply:
Then you also don't need the `in_guild` ward because it is dependent.

Reply:
meow !